SphinxDocs/source/Manual/D.rst

```diff
@@ -333,8 +335,8 @@ When generating D wrappers, a few additional macros are available:
 
 ``$dcall`` and ``$winput`` (C#: ``$cscall``, ``$iminput``)
    These variables are used in the director-specific typemaps
-   ```ddirectorin`` <D.html#D_ddirectorinout>`__ and
-   ```ddirectorout`` <D.html#D_ddirectorinout>`__. They are more or less
+   :ref:`ddirectorin <D_ddirectorinout>` and
+   :ref:`ddirectorout <D_ddirectorinout>`. They are more or less
```

Comment:
When I run docdiffer.sh, this change results in:
```
...
+ rst2html.py ../SphinxDocs/source/Manual/D.rst
../SphinxDocs/source/Manual/D.rst:339: (ERROR/3) Unknown interpreted text role "ref".
../SphinxDocs/source/Manual/D.rst:339: (ERROR/3) Unknown interpreted text role "ref".
```
Firstly, I don't really know what this is about as the link seems to work. Secondly, I don't see this error when I run `make html`. Why not?

Reply:
Using an extra space solves the problem. I pushed the changes.
In `docdiffer.sh` you are using `rst2html` whereas in make you are using `sphix`. Both interpret the input differently. That's the reason `make.html` didn't give any error.

Reply:
now the output of rst2html is wrong, surely this should work properly. Why did you use `:ref:` and not the normal linking syntax?

Reply:
@manitejav ???

Reply:
@wsfulton For internal linking you need :ref: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations.